docs/protocol/life_cycle_of_a_service.md

```diff
@@ -0,0 +1,41 @@
+The life cycle of a service registered in the Autonolas Protocol passes through the following states:
+
+* pre-registration,
+* active registration,
+* finished registration,
+* deployed, and
+* terminated bonded.
+
+You can see the current state of a service by browsing the [services section](https://protocol.autonolas.network/services) in the frontend, and clicking on the corresponding service.
+
+<figure markdown>
+![Life cycle of a service](images/life_cycle_of_a_service.svg){ align=left width=600 }
+</figure>
+
+Only the service owner can change the state of a given service. Connect your wallet to the frontend in order to make changes.
```

Comment:
This is not entirely correct. `Finished-registration` changes by the last operator registering agent instances to their full capacity.

Reply:
fixed